src/analysis/convex/function.lean

```diff
@@ -104,6 +104,29 @@ lemma strict_concave_on.subset {t : set E} (hf : strict_concave_on 𝕜 t f) (hs
   strict_concave_on 𝕜 s f :=
 ⟨hs, λ x y hx hy, hf.2 (hst hx) (hst hy)⟩
 
+section composition
+variables Q : Type*
+variables [has_smul 𝕜 Q] [ordered_add_comm_monoid Q]
+variables (t : set β) (g : β → Q)
```

Comment:
I don't think it's very appropriate to use a `variables` block for a single theorem, but since this is done elsewhere in the file, I can give it a pass. If you choose to keep them, you should merge these `variables` blocks into one.